CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 5.1.1 - 2020-05-15
+
+This release contains minor deprecations of names of methods and classes to make this library more idiomatic with the wider JavaScript ecosystem.
```

Comment:
I thought this was a Java SDK? (Says JavaScript here)

Reply:
You and me both :) 

Fixed